go.mod

```diff
@@ -1,20 +1,60 @@
 module github.com/microsoft/frameworkcontroller
 
-go 1.12
+go 1.18
 
 require (
-	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
+	github.com/pkg/errors v0.9.1
```

Comment:
Why much more dependencies now? Expected? Could we remove some?

Reply:
This is code-gen from `go mod tidy`. Let me see if there is less